Add list operation example (#1183)
contao/docs: docs/dev/reference/dca/callbacks.md

**return:** `string` HTML for the button
{{% /expand %}}

{{% expand "Example" %}}

This example hide a custom operation button if the user is not allowed to use it.    

> Attention: this won't disable the operation itself, it only hides the button!
> To disable the operation, you need to check for the permission additional before it is execution, for example in the operation backend module code.

Reviewer: ```suggestion
Attention: this won't disable the operation itself, it only hides the button!
To disable the operation, you need to check for the permission additionally 
before its execution, for example in the operation code or a `config.onload` callback.
```
